src/gl-state-wgl.cpp

```diff
@@ -133,7 +141,7 @@ GLStateWGL::swap()
 }
 
 bool
-GLStateWGL::gotNativeConfig(int& vid)
+GLStateWGL::gotNativeConfig(intptr_t& vid)
```

Comment:
Is there some Windows goof that makes this necessary?  We're not passing pointers here.

Reply:
See the base function here:

https://github.com/glmark2/glmark2/blob/master/src/gl-state.h#L40

I think this was changed recently by an upstreamed ChromeOS CL.

Reply:
Yes, the signature was updated to accommodate some ChromeOS related needs. I have split out the WGL fix and merged it independently here: https://github.com/glmark2/glmark2/commit/c17fd14505f30d9e4dbad276f7aa956fd21a637b

Reply:
Thanks Alexandros. I rebased this PR on top of master.